### ISSUE TITLE
Add all of our Prod & Test articles to Optimo Test

### ISSUE THREAD

**Issue by sareh** [bbc/simorgh#764]
**Is your feature request related to a problem? Please describe.**
We should add our News Articles from Prod and Test into the Optimo TEST CMS.

This is so that we can use them for our Cypress tests and for testing in general when we are getting data from Optimo. 

**Describe the solution you'd like**
We should ensure we upload the correct images with the variety of ratios, according to this document: https://github.com/BBC-News/simorgh/blob/latest/data/README.md

When we have created the Optimo articles on their TEST environment, we should add a column in the table on that link that has the correct Optimo id `cxxxxxxxxxxo`. 

**Describe alternatives you've considered**
N/A

**Testing notes**
Manually checking whether the data feed provided to us at `$ARES_BROKER/api/article/:id` for each of the 1-28 articles, looks correct. 

**Additional context**
N/A

**Comment by sareh** [comment]
This is also blocked, pending Optimo fixing their image uploader. 
After that is addressed, then this ticket should be blocked until we finish this ticket:  https://github.com/BBC-News/simorgh/issues/736.

**Comment by jimjohnsonrollings** [comment]
Fixture Scenario | Test Optimo ID | Published | Notes
------------ | ------------- | ------------ | ----------
c0000000001o | cnm5l5djlgxo | 👍 | 
c0000000002o | c5mk3kl3w3ko| 👍 | 
c0000000003o | c6eplpm44vlo | 👍 | 
c0000000004o | c4vzwzjp2m6o | 👍 | 
c0000000005o | cq80n03rj0qo  | 👍 | 
c0000000006o | c5m4y4y36kko | 👍 | 
c0000000007o | c1d36dmmry5o | 👍  |
c0000000008o | cge525m34kzo | 👍 | Missing image caption as cannot publish image captions with urlLinks in
c0000000009o | cj3g8gymk66o | 👍 |
c0000000010o | c7zj2z16p7jo | 👍  |
c0000000011o | c457lnly75po | 👍 | 
c0000000012o | cq3g5kd10v1o | 👍 | 
c0000000013o | cr102p9d55eo | 👍 | 
c0000000014o | c91n655l2r1o | 👍 |
c0000000015o | c0r60n52n51o | 👍 | 
c0000000016o | ck6l5p9eg22o | 👍 | 
c0000000017o | cq3gdp3zkm5o | 👍 | 
c0000000018o | c91n6510z7wo | 👍  | 
c0000000019o | c457ypqmmzeo | 👍 | 
c0000000020o | cgq4r501egwo | 👍 | Image caption does not contain urlLink
c0000000021o | cjd8zd49d4qo | 👍 |
c0000000022o | cw2qrpznzqvo | 👍 | 
c0000000023o | cpz770qwzypo | 👍 | Withdrawn article
c0000000024o | c5nd2n466ppo | 👍 |
c0000000025o | c9rpqy7pmypo | 👍 |
c0000000026o | cd2dyze7347o | 👍 | 
c0000000027o | c85pqyj5m2ko | 👍 |
c0000000028o | cwv2xv848j5o | 👍 |

Other notes
- Subscript & Superscript are not currently available in Optimo
- The videos used in the original scenarios aren't available on Test, so alternatives have been used
- Summary is not yet available in Optimo

**Comment by jimjohnsonrollings** [comment]
Blocked again as Optimo currently can't publish images.

**Comment by jtart** [comment]
Optimo can now publish images and inline links.

**Comment by jimjohnsonrollings** [comment]
Blocked as Optimo is struggling to publish

**Comment by jimjohnsonrollings** [comment]
Unblocked again

**Comment by jimjohnsonrollings** [comment]
Further work blocked until Optimo can support urlLinks in image captions, superscript and/or subscript.

**Comment by jimjohnsonrollings** [comment]
Follow-up issue #1005 exists to complete the missing parts of this once Optimo makes the features available
